crates/daphne/src/testing/report_generator.rs

```diff
@@ -75,6 +67,7 @@ impl ReportGenerator {
         version: DapVersion,
         now: Time,
         extensions: Vec<messages::Extension>,
+        replay_reports: bool,
```

Comment:
Does this mean "replay reports" or "disable replay protection"? If the latter, then change the type to `ReplayProtection`.

Reply:
it means "replay reports"